_posts/course_reviews/third_year/2020-12-26-AE-333.md

```diff
@@ -5,8 +5,65 @@ aside:
   toc: true
 sidebar:
   nav: docs-en
-permalink: /course_reviews/third_year/AE-333.html
+permalink: /course_reviews/third_year/ae-333.html
```

Comment:
Check the permalink and categories thing from AE 624 and keep the format the same.